tests/unit/local/lambdafn/test_runtime.py

```diff
@@ -61,10 +61,7 @@ def test_must_create_lambda_container(self, LambdaContainerMock):
 
         LambdaContainerMock.return_value = container
 
-        self.runtime.create(self.func_config, debug_context=debug_options, event=event)
-
-        # Verify if Lambda Event data is set
-        self.env_vars.add_lambda_event_body.assert_called_with(event)
+        self.runtime.create(self.func_config, debug_context=debug_options)
```

Comment:
Same here, remove the `event` var created on line 49

Reply:
resolved